rfc8446.xml

```diff
@@ -109,84 +104,111 @@ compatible with previous versions, all versions of TLS incorporate a
 
 <t>The following is a list of the major functional differences between
-TLS 1.2 and TLS 1.3. It is not intended to be exhaustive and there
+TLS 1.2 and TLS 1.3. It is not intended to be exhaustive, and there
 are many minor differences.</t>
 
 <t><list style="symbols">
   <t>The list of supported symmetric algorithms has been pruned of all algorithms that
 are considered legacy. Those that remain all use Authenticated Encryption
-with Associated Data (AEAD) algorithms. The ciphersuite concept has been
+with Associated Data (AEAD) algorithms. The cipher suite concept has been
 changed to separate the authentication and key exchange mechanisms from
 the record protection algorithm (including secret key length) and a hash
-to be used with the key derivation function and HMAC.</t>
-  <t>A 0-RTT mode was added, saving a round-trip at connection setup for
+to be used with the key derivation function and the Hashed Message
+Authentication Code (HMAC).</t>
+  <t>A zero round-trip time (0-RTT) mode was added, saving a round trip at connection setup for
 some application data, at the cost of certain security properties.</t>
   <t>Static RSA and Diffie-Hellman cipher suites have been removed;
 all public-key based key exchange mechanisms now provide forward secrecy.</t>
   <t>All handshake messages after the ServerHello are now encrypted. The
-newly introduced EncryptedExtension message allows various extensions
-previously sent in clear in the ServerHello to also enjoy
+newly introduced EncryptedExtensions message allows various extensions
+previously sent in the clear in the ServerHello to also enjoy
 confidentiality protection from active attackers.</t>
-  <t>The key derivation functions have been re-designed. The new design allows
+  <t>The key derivation functions have been redesigned. The new design allows
 easier analysis by cryptographers due to their improved key separation
 properties. The HMAC-based Extract-and-Expand Key Derivation Function (HKDF)
 is used as an underlying primitive.</t>
   <t>The handshake state machine has been significantly restructured to
 be more consistent and to remove superfluous messages such as
 ChangeCipherSpec (except when needed for middlebox compatibility).</t>
-  <t>Elliptic curve algorithms are now in the base spec and new signature
+  <t>Elliptic curve algorithms are now in the base spec, and new signature
 algorithms, such as ed25519 and ed448, are included. TLS 1.3 removed point format
 negotiation in favor of a single point format for each curve.</t>
-  <t>Other cryptographic improvements including the removal of compression and
-custom DHE groups, changing the RSA padding to use RSASSA-PSS, and the removal of
-DSA.</t>
+  <t>Other cryptographic improvements, including the removal
+of compression, the Digital Signature Algorithm (DSA), and custom Ephemeral
+Diffie-Hellman (DHE) groups, changing the RSA padding to use the RSA
+Probabilistic Signature Scheme (RSASSA-PSS), have been added.
+</t>
```

Comment:
This paragraph is a bit awkward, (the "and" isn't in the right place, or rather, there are two nested sets of "and") consider revising to:

```
-Diffie-Hellman (DHE) groups, changing the RSA padding to use the RSA
+Diffie-Hellman (DHE) groups, and changing the RSA padding to use the RSA
```

Or list the removed features in a parenthetical:

> Other cryptographic improvements have been made, including
> the removal of insecure features (compression, the Digital
> Signature Algorithm (DSA), and custom Ephemeral Diffie-Hellman
> (DHE) groups) and changing the RSA padding to use the RSA
> Probabilistic Signature Scheme (RSASSA-PSS).